index.html

```diff
@@ -1,59 +1,284 @@
+          </div>
+        </article>
+
+        <article class="project-articles">
+          <div class="textside">
+            <h2>Tonic</h2>
+            <ul class="card-ul">
+              <li>CANOPY</li>
+              <li><img src="./images/counter.png" /></li>
+              <li>Back End Dev</li>
+              <li><img src="./images/counter.png" /></li>
+              <li>2015</li>
+            </ul>
+            <p>
+              Experiment content creation feature that allows users to add an
+              existing story over the course of a day without spamming their
+              friends
+            </p>
+            <ul class="skills">
+              <li>html</li>
+              <li>Ruby on</li>
+              <li>css</li>
+              <li>javascript</li>
+            </ul>
+            <button class="see-project">See Project</button>
+          </div>
+          <div class="card">
+            <img src="./images/snapshot4.png" />
+          </div>
+        </article>
+      </section>
+
+      <section id="about">
+        <div>
+          <h2>About<br />Myself</h2>
+          <p>
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+          <div class="social-media">
+            <a href="#" class="social-para social-blue">LET'S CONNECT</a>
+            <ul>
+              <li>
+                <a
+                  target="_blank"
+                  class="social-tag"
+                  href="https://www.github.com/joeveoper"
+                  ><img src="./images/twitter.png"
+                /></a>
+              </li>
+              <li>
+                <a
+                  target="_blank"
+                  class="social-tag"
+                  href="https://www.linkedin.com/in/yinka-olatunji-83aa36b5/"
+                  ><img src="./images/linkedin.png"
+                /></a>
+              </li>
+              <li>
+                <a
+                  target="_blank"
+                  class="social-tag"
+                  href="https://www.twitter.com/joeveoper"
+                  ><img src="./images/medium.png"
+                /></a>
+              </li>
+              <li>
+                <a
+                  target="_blank"
+                  class="social-tag"
+                  href="https://www.medium.com/"
+                  ><img src="./images/github.png"
+                /></a>
+              </li>
+              <li>
+                <a
+                  target="_blank"
+                  class="social-tag"
+                  href="https://www.angellist.com/"
+                  ><img src="./images/angellist.png"
+                /></a>
+              </li>
+            </ul>
+            <button class="see-project">Get My Resume</button>
```

Comment:
Nice work building the about section of the template👍 But we can still do better in sticking to the design😉 Kindly take note of the highlighted parts😉 and ensure to stick to the design as you make your changes. take note of the spacing around every component, their font properties, and their colors😉

design
<img width="235" alt="design about" src="https://user-images.githubusercontent.com/64643995/220053690-39a1f54a-2f40-4d28-b5bd-5b0e2aa750c1.png">

yours
<img width="319" alt="about" src="https://user-images.githubusercontent.com/64643995/220054220-b62d9350-33bb-4283-95a4-416d3fbe5913.png">